lib/AnyEvent/WebSocket/Client.pm

```diff
@@ -19,11 +19,11 @@ use PerlX::Maybe qw( maybe provided );
 =head1 SYNOPSIS
 
  use AnyEvent::WebSocket::Client 0.12;
- 
```

Comment:
Please do not remove this space from the POD verbatim blocks.  Without the indentation, some POD renderers break these into separate verbatim blocks and it makes the synopsis and other code examples look bad.

Reply:
So?  no pod testing eh?  meh..

OK, For the record I have reverted the broken stuffs..
Here is what we get when we validate the pod syntax..


*** WARNING: line containing nothing but whitespace in paragraph at line 37 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 40 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 48 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 55 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 59 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 154 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 446 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 448 in file lib/AnyEvent/WebSocket/Client.pm
*** WARNING: line containing nothing but whitespace in paragraph at line 454 in file lib/AnyEvent/WebSocket/Client.pm
lib/AnyEvent/WebSocket/Client.pm pod syntax OK.